ee/clickhouse/models/cohort.py

```diff
@@ -81,13 +87,31 @@ def parse_action_timestamps(days: int) -> Tuple[str, Dict[str, str]]:
 
 
 def format_filter_query(cohort: Cohort) -> Tuple[str, Dict[str, Any]]:
-    person_query, params = format_person_query(cohort)
+    person_query, params = determine_precalculated_or_live_person_query(cohort)
     person_id_query = CALCULATE_COHORT_PEOPLE_SQL.format(
         query=person_query, latest_distinct_id_sql=GET_LATEST_PERSON_DISTINCT_ID_SQL
     )
     return person_id_query, params
 
 
+def determine_precalculated_or_live_person_query(cohort: Cohort) -> Tuple[str, Dict[str, Any]]:
+    if (
+        cohort.last_calculation
+        and cohort.last_calculation > TEMP_PRECALCULATED_MARKER
```

Comment:
we won't need this marker once all existing cohorts have done a single passthrough of recalculating